content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,105 @@ menu:
+- Ethernet or Wi-Fi connection between a DHCP-enabled network and the Pi
+- Personal computer(Linux/MacOS recommended) on the same network as the Pi
+
+## Setting up the Raspberry Pi
+1. Set up your SD card as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/2)
+2. Connect your Raspberry Pi as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/3)
+3. Start up your Raspberry Pi as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/4)
+4. Configuring your Raspberry Pi described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-using/9)
+    
+    #### Configure SSH(Remote Access Setting)
+    You make sure to enable SSH to allow remote access to your Raspberry Pi from another computer using SSH. 
+    
+    Simple way to enable SSH as described [here](https://www.raspberrypi.org/documentation/remote-access/ssh/README.md) is to select `Enable` next to `SSH` in the interface tap of `Raspberry Pi configuration` from the `Preferences` menu.
+    
+    ![Interfaces] (/images/quickstart/pi-configuration-interfaces.png)
+    
+    Or 
+    
+    You can also configure SSH via `raspi-config` on the Pi terminal. 
+    ```shell script
+    sudo raspi-config
+   
+    # Select 'Interfacing Options'
+    # Navigate to and select 'SSH'
+    # Choose 'Yes'
+    # Select 'Ok'
+    # Choose 'Finish'
+    ```
+
+## Connect to the Raspberry Pi via SSH described [here](https://www.raspberrypi.org/documentation/remote-access/ssh/unix.md)
+1. Open a terminal on your computer.  
+2. Get IP address of the Raspberry Pi board by following command on the terminal. 
+
+    ```
+    ping raspberrypi.local 
+    ```
+    If the ping command doesn't reach raspberrypi.local due to DNS issues, you can connect a keyboard and monitor to the Raspberry Pi and run `ifconfig` command in the terminal.
+
+3. Connect to the Raspberry Pi via SSH.
+
+    For Linux or macOS, use either of these commands:
+    ```shell script
+    # replace with current hostname if changed.
+    ssh pi@raspberrypi.local  
+    ```
+    Or 
+    ```shell script
+    # use IP address from previous step
+    ssh pi@ipaddress  
+    ```
+4. Type the password when prompted. Default login username is `pi` and password is `raspberry`.
+
+## Install essential build tools
+1. Update & upgrade all the installed packages.
+    ```shell script
+    sudo apt-get update
+    sudo apt-get upgrade
+    ```
+2. Install build tools and so on.
+    ```shell script
+    sudo apt-get install build-essential git curl openssl vim jq tac awk
+    ```
+
+## Install cloud server (OCF cloud enabled device) 
+1. Download IoTivity-Lite open soure from the GitHub. 
+
+    Continue on your personal PC, from the SSH prompt, go to the home directory and clone the latest open source of IoTivity-Lite.
+    
+    ```shell script
+    git clone https://gitlab.iotivity.org/iotivity/iotivity-lite.git
```

Comment:
--recursive not needed?

Reply:
Yes, no need to add  '--recursive' but I will add as you commented.